docs/getting_started/installation.rst

```diff
@@ -136,6 +136,22 @@ install the assets:
     bin/console cache:clear
     bin/console assets:install
 
+Webpack Encore Integration
+--------------------------
+
+When the assets are installed add following to ``config/webpack_encore.yaml``
```

Comment:
Maybe we could suggest to configure this in `config/sonata_admin.yaml` instead.

Reply:
Yes 👍🏻